MCprep_addon/world_tools.py

```diff
@@ -19,13 +19,17 @@
 import os
 import math
 from pathlib import Path
+from typing import List, Optional
 import shutil
 
 import bpy
+from bpy.types import (
+  Context, Object, Camera
+)
```

Comment:
Just like with past suggestions
```suggestion
from bpy.types import (
  Context, Camera
)
```